easybuild/easyconfigs/j/Jorg/Jorg-1.0.0-foss-2020b.eb

```
@@ -0,0 +1,38 @@
+# Author: Ben Langenberg ben.langenberg@ufz.de
+# Helmholtz-Centre for Environmental Research
+# Leipzig - Germany
+
+easyblock = 'PackedBinary'
+
+name = 'Jorg'
+version = '1.0.0'
+
+homepage = 'https://github.com/lmlui/Jorg'
+description = "A MAG Circularization Method By Lauren Lui, Torben Nielsen, and Adam Arkin"
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+sources = [{
+    'filename': '%(name)s-%(version)s.tar.gz',
+    'git_config': {
+        'url': 'https://github.com/Helmholtz-UFZ',
+        'repo_name': 'Jorg',
+        'tag': 'v1.0.0',
+    },
+}]
```

@DirectorSloan `git_config` should only be used if needed, because it does not allow checksums to be used: checksum is not garanteed to be the same because Easybuild itself generates the tarball.

In this case it's not needed:
```suggestion
github_account = 'Helmholtz-UFZ'
source_urls = [GITHUB_SOURCE]
sources = ['v%(version)s.tar.gz']
checksums = ['74b1c455ba57ab240921966255ca059840b97c47efcd16e9e66b9b6a8a136ee2']
```